nio/exceptions.py

```diff
@@ -47,6 +47,12 @@ class OlmTrustError(Exception):
     pass
 
 
+class OlmUnverifiedDeviceError(OlmTrustError):
+    def __init__(self, *args, unverified_device=None) -> None:
```

Comment:
The `-> None` is still invalid syntax for python2. If you want type annotations you'll have to use them in the comments like the rest of the code does. At least until we drop py2 support.